docs/src/data/nav-items.yaml

```diff
@@ -20,6 +20,8 @@
       path: /guides/CreateFHIRValidationAssembly
     - title: fhir-search configuration
       path: /guides/FHIRSearchConfiguration
+    - title: Bring your own Persistence
+      path: /guides/BYOPersistence
```

Comment:
seems better to use full word ...

Reply:
"BYO" is a common acronym (typically a play on "BYOB") and its only in the URL, but i don't feel strongly about keeping it